### ISSUE TITLE
ElasticSearch Error Code 78

### ISSUE THREAD

**Issue by edsu** [DocNow/docnow#99]
Jane Kelly reports in the DocNow Slack that ElasticSearch exits when doing a `docker-compose up`:

    docnow_elasticsearch_1 exited with code 78

She is running the DocNow app inside a VirtualBox VM running xubuntu 18.04.01. The VM base memory was initially set to 2731MB and was upped to 4000MB with no change.

The plan is to reproduce this problem on my own workstation using VirtualBox and go from there.

**Comment by kayiwa** [comment]
heya @edsu I encountered the same problem on a linux VM and also while troubleshooting with terraform. Have them try inside the host VM as `root`

```
sysctl -w vm.max_map_count=262144
```
It worked for me.

**Comment by edsu** [comment]
Ahah, that looks very similar to what @raffazizzi had to do on his host Linux system. We should ping Jane and have her give it a try -- and perhaps document in the docnow README?

**Comment by kayiwa** [comment]
what is not immediately clear is this runs before we see the `exited with code 78`
```bash
initial heap size [268435456] not equal to maximum heap size [2147483648]; this can cause resize pauses and prevents mlockall from locking the entire heap
max virtual memory areas vm.max_map_count [65530] likely too low, increase to at least [262144]
```

**Comment by kayiwa** [comment]
> Ahah, that looks very similar to what @raffazizzi had to do on his host Linux system. We should ping Jane and have her give it a try -- and perhaps document in the docnow README?

PR for README coming later and also moving to Elastic 6.6 container when I've made sure it works on macOS.

**Comment by edsu** [comment]
I think this is a warning, that the JVM will have to "stop the world" while it allocates more heap space. Possibly an issue for anyone using Docker in production.

**Comment by edsu** [comment]
@kayiwa has fixed this in #100 by adding a note for Ubuntu users where ElasticSearch fails to start because the Docker host can't allocate enough memory. 

```
sysctl -w vm.max_map_count=262144
```
